### PR TITLE
build: Release v30.0

### PR DESCRIPTION
**Pull request by likebreath**
Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ homepage = "https://github.com/cloud-hypervisor/cloud-hypervisor"
 # Keep in sync with version in .github/workflows/build.yaml
 # Policy on MSRV (see #4318):
 # Can only be bumped by:
-# a.) A dependency requires it, 
+# a.) A dependency requires it,
```
Comment (rbradford) [inline Cargo.toml:15]: nit: unrelated change?

**Review by rbradford — COMMENTED** [review]
There are quite a few extra changes in the release notes file. Perhaps you should exclude those.

File: Cargo.toml (hunk 1 above)
Comment (likebreath) [reply Cargo.toml:15]: I separate the changes of removing whitespaces into its own commit. PTAL.

**Review by rbradford — APPROVED** [review]
